.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
IDE-specific files shouldn't be committed to version control, so please remove it.